Representable/representable
# state resources pages

**changes**
- adds a dropdown to resources page for state selection
- adds urls for state specific resources page -- on these pages, see state specific redistricting criteria, coi definitions, and process for all 50 states

**to test**
- python manage.py runserver
- go to resources page
- use dropdown
- check behavior is as expected

**screenshots**
regular resources page:
![image](https://user-images.githubusercontent.com/41943646/127037364-9f898698-a594-4664-a6c5-184e63e344a9.png)
with state info
![image](https://user-images.githubusercontent.com/41943646/127037399-4fac36b5-4ebf-4436-b83f-21784beabe1c.png)
